### PR TITLE
Add points for point systems

### PR DESCRIPTION
**Pull request by ghost**
I added points to MSGIDs and Origin Lines for point systems. Previously it was using the non point address even if the system was configured as a point.

Also has the math.h in wwivd to make it compile

### CODE REVIEW

File: sdk/fido/fido_util.cpp [hunk 1]
```diff
@@ -189,6 +189,10 @@ std::string to_zone_net_node(const wwiv::sdk::fido::FidoAddress& a) {
   return StrCat(a.zone(), ":", to_net_node(a));
 }
 
+std::string to_zone_net_node_point(const wwiv::sdk::fido::FidoAddress& a) {
```
Comment (wwiv) [inline sdk/fido/fido_util.cpp:192]: Seems like I should just call that to_string and always include the point - @apamment Did you notice a place where we don't want the point included (I know little of FTN)

File: init/wwivd_ui.cpp [hunk 2]
```diff
@@ -21,7 +21,7 @@
 #include <cstdint>
 #include <memory>
 #include <string>
-
+#include <math.h>
```
Comment (wwiv) [inline init/wwivd_ui.cpp]: Did this cause a compile error?

usually including C only headers we do as:

#include <cmath>

**Comment by wwiv** [comment]
Thanks @apamment  - I've not tried anything with points, so more things may be broken.

**Comment by MarkHofmann11** [comment]
Thanks, @apamment!  It is great to see you working with Rushfan on things!
I know you have done lots (getting WWIVnet working on your custom BBS for
example).  That was a first!

File: sdk/fido/fido_util.cpp (hunk 1 above)
Comment (ghost) [reply sdk/fido/fido_util.cpp:192]: I'm not a guru at ftn either, but I *think* the msgid and origin line are only supposed to have a point if it's not zero.

**Pull request merged**